src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -92,15 +111,41 @@ public class WwwAuthenticateParameters
         {
             if (httpResponseHeaders.WwwAuthenticate.Any())
             {
-                // TODO: add POP support
-                AuthenticationHeaderValue bearer = httpResponseHeaders.WwwAuthenticate.First(v => string.Equals(v.Scheme, scheme, StringComparison.OrdinalIgnoreCase));
+                AuthenticationHeaderValue bearer = httpResponseHeaders.WwwAuthenticate.First(v => string.Equals(v.Scheme, Constants.BearerAuthHeaderPrefix, StringComparison.OrdinalIgnoreCase));
```

Comment:
I don't see why this method can't handle any authentication scheme out there. it's just parsing logic.

Reply:
It can, this is just a bug left over from testing/refactoring